### PR TITLE
liquibase management implemented, user and role models provided v6.0

### PR DESCRIPTION
**Pull request by CodeWebWeaver**

### CODE REVIEW

File: src/main/java/com/parkhomovsky/bookstore/exception/RegistrationException.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.parkhomovsky.bookstore.exception;
+
+public class RegistrationException extends RuntimeException {
+    public RegistrationException(String message) {
+        super(message);
```
Comment (Someboty) [inline src/main/java/com/parkhomovsky/bookstore/exception/RegistrationException.java]: Maybe RegistrationException should be extended by Exception

File: src/main/resources/db/changelog/changes/03=create-users-table.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                    nullable: false
+              - column:
+                  name: shipping_address
+                  type: varchar(255)
+              - column:
+                  name: is_deleted
+                  type: boolean
+                  defaultValueBoolean: false
+                  constraints:
+                    nullable: false
```
Comment (Someboty) [inline src/main/resources/db/changelog/changes/03=create-users-table.yaml]: You lost an empty line here

File: src/main/resources/db/changelog/changes/04-create-roles-table.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: name
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+                    unique: true
```
Comment (Someboty) [inline src/main/resources/db/changelog/changes/04-create-roles-table.yaml]: And here

File: src/main/resources/db/changelog/changes/05-create-users-roles-table.yaml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+                  type: bigint
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: role_id
+                  type: bigint
+                  constraints:
+                    primaryKey: true
+                    nullable: false
```
Comment (Someboty) [inline src/main/resources/db/changelog/changes/05-create-users-roles-table.yaml]: And here?

File: src/main/resources/db/changelog/changes/06-test-insert-into-roles-table.yaml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+            columns:
+              - column:
+                  name: name
+                  value: ROLE_ADMIN
+        - insert:
+            tableName: roles
+            columns:
+              - column:
+                  name: name
+                  value: ROLE_USER
```
Comment (Someboty) [inline src/main/resources/db/changelog/changes/06-test-insert-into-roles-table.yaml]: Looks like you don't like them)

File: src/main/resources/db/changelog/changes/07-test-insert-in-users-table.yaml [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+                  value: I$J(QJ#T(QIG#Q*G(Q#JIRARJAK@(JRHI==wf
+        - insert:
+            tableName: users_roles
+            columns:
+              - column:
+                  name: user_id
+                  valueNumeric: 2
+              - column:
+                  name: role_id
+                  valueNumeric: 2
```
Comment (Someboty) [inline src/main/resources/db/changelog/changes/07-test-insert-in-users-table.yaml]: But give them a chance)

**Review by Someboty — APPROVED** [review]
Great job! I leave a few propositions here.

**Review by vovasalyha — APPROVED** [review]
GJ

**Pull request merged**